packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -181,6 +185,21 @@ export class AtomicResultList implements InitializableComponent {
           content={template.innerHTML}
         ></atomic-result>
       );
+
+      return this.display === 'grid' ? (
```

Comment:
As soon as the results are displayed grid, any click on the "card" will trigger the default behaviour of opening the result.